frontend/src/components/History/index.jsx

```diff
@@ -178,7 +183,7 @@ class History extends React.Component {
     }
 
     render() {
-        const { history_data } = this.props;
+        const { history_data, unlabeled_data } = this.props;
```

Comment:
So I think for the use case we generally see here, it might make more sense to merge these two lists on the backend, sort alphabetically, and then return. It seems like they're going to be comparing similar data that is both labeled and unlabeled side by side.